RecoTracker/LST/python/lstProducer_cff.py

```diff
@@ -7,3 +7,6 @@
 
 from RecoTracker.LST.lstModulesDevESProducer_cfi import lstModulesDevESProducer
 (~gpu).toModify(lstModulesDevESProducer.alpaka, backend = 'serial_sync')
+
+# not scheduled task to get the framework to hide the producer
+dummyLSTModulesDevESProducerTask = cms.Task(lstModulesDevESProducer)
```

Comment:
@makortel
does this look OK?
This way I don't need to make the full multi-module LST-related task visible in RecoTracker/IterativeTracking/python/HighPtTripletStep_cff.py